### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by Tonio-an**

### CODE REVIEW

File: src/main/java/core/basesyntax/Plane.java [hunk 1]
```diff
@@ -8,4 +8,48 @@
...
+    private int humanCapacity;
+    private int maxSpeed;
+    private int engineAmount;
+
+    private Plane(PlaneBuilder planeBuilder) {
+    }
+
+    public static void main(String[] args) {
+
+    }
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Plane.java]: what's main doing here?

File: src/main/java/core/basesyntax/Plane.java [hunk 2]
```diff
@@ -8,4 +8,48 @@
 
 public class Plane {
 
+    private String brand;
+    private String model;
+    private int humanCapacity;
+    private int maxSpeed;
+    private int engineAmount;
+
+    private Plane(PlaneBuilder planeBuilder) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Plane.java]: why is this constructor empty?

File: src/main/java/core/basesyntax/Plane.java [hunk 3]
```diff
@@ -8,4 +8,48 @@
...
+    }
+
+    public static void main(String[] args) {
+
+    }
+
+    public class PlaneBuilder {
+        public PlaneBuilder setBrand(String brand) {
+            Plane.this.brand = brand;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Plane.java]: you should set builder's fields

File: src/main/java/core/basesyntax/Plane.java [hunk 4]
```diff
@@ -8,4 +8,48 @@
...
+            return this;
+        }
+
+        public PlaneBuilder setEngineAmount(int engineAmount) {
+            Plane.this.engineAmount = engineAmount;
+            return this;
+        }
+
+        public PlaneBuilder build() {
+            return PlaneBuilder.this;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Plane.java]: I can't see `building` process :)

File: src/main/java/core/basesyntax/Plane.java [hunk 5]
```diff
@@ -7,5 +7,56 @@
...
+        private int maxSpeed;
+        private int engineAmount;
+
+        public PlaneBuilder setBrand(String brand) {
+            Plane.this.brand = brand;
+            return this;
+        }
+
+        public PlaneBuilder setModel(String model) {
+            Plane.this.model = model;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/Plane.java]: to set builders field do we need to write Plane?

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**